public/assets/ts/editor-loader.ts

```diff
@@ -1,23 +1,17 @@
+interface JSONResponse {
+  endpoint: string;
+  response: any;
```

Comment:
hmm thoughts about doing front end type checking as well? or do you think it's a lil overkill

Reply:
like we could also do front end schema validations

Reply:
for a future pr *

Reply:
Seems overkill - since we already have backend error checking, we can send an error to the frontend if something's wrong.  Is there something frontend validation would add?

Reply:
ah ig you're right, if the backend always validates the schema then there's really no chance of the frontend getting a bad object